resolwe_bio/tests/files/test_metadata_upload/inputs/sample_metatable.tsv

```diff
@@ -0,0 +1,11 @@
+function	gene	spo-early	spo-mid	heat 0	heat 10	heat 20
```

Comment:
Use the new Orange format for this use case.

Reply:
Thanks this is important to test as well.